app/src/main/java/com/dongyang/android/youdongknowme/data/remote/entity/Cafeteria.kt

```diff
@@ -3,16 +3,8 @@ package com.dongyang.android.youdongknowme.data.remote.entity
 import com.google.gson.annotations.SerializedName
 
 data class Cafeteria(
-    @SerializedName("range")
-    var range: String,
     @SerializedName("date")
     var date: String,
-    @SerializedName("restaurant")
-    var restaurant: String,
-    @SerializedName("menu_division")
-    var menuDivision: String,
-    @SerializedName("menu_content")
-    var menuContent: String,
-    @SerializedName("etc_info")
-    var etcInfo: String,
+    @SerializedName("menus")
+    val menu: List<String>
```

Comment:
```suggestion
    val menus: List<String>
```

가독성을 위해 컬렉션은 복수형태로 이름 지어주시면 좋을 것 같아요~

Reply:
메뉴 관련 브랜치를 따로 제작하여 위에 `stu` 수정과 같이 수정하는게 좋을거 같습니다.